src/js/state/Current/selectors.ts

```diff
@@ -51,9 +52,17 @@ export const getQuery = (state: State): BrimQuery | null => {
     Queries.getQueryById(queryId)(state) ||
     RemoteQueries.getQueryById(queryId)(state)
   if (!query) return null
-  return new BrimQuery(query)
```

Comment:
this was the culprit of all of the re-rendering, since it always creates a new object